src/styles/theme.scss

```diff
@@ -128,6 +128,17 @@ h5 {
   color: $text-colour;
 }
 
+.icon-arrow-right {
+  /* content: "\e901"; */
```

Comment:
if you change this to `""` and leave the icon css intact, is that the same effect? Do you also need to 0 height/ width?

Reply:
`""` indeed has the same effect. now, this css for the icon is normally in icon.css but I moved it here for the custom-built icon. if you want me to use an svg icon from icomoon it's 2 minutes to change.

`0 height and width` are indeed not impacting that but I've seen it applied in all the examples I looked at so... :man_shrugging: